.github/scripts/update-manifests-tags.sh

```
@@ -0,0 +1,17 @@
+
+set -euo pipefail
+
+update_tags(){
+    sed  -i -r "/\"$1\"/s|([^:]*):([^:]*):[^:]*:(.*)|\1:\2:$2:\3|" get_all_manifests.sh
+}
+
+prefix=component_spec_
+
+echo
+env | while IFS="=" read varname value; do
+    [[ $varname =~ "${prefix}" ]] || continue
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)